webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-summary.js

```diff
@@ -0,0 +1,363 @@
+import {RubricsElement} from "./rubrics-element.js";
+import {html} from "/webcomponents/assets/lit-element/lit-element.js";
+import "./sakai-rubric-sort-name.js";
+
+export class SakaiRubricSummary extends RubricsElement {
+
+    constructor() {
+        super();
+    }
+
+    static get properties() {
+        return {
+            token: String,
+            toolId: { attribute: "tool-id", type: String },
+            entityId: { attribute: "entity-id", type: String },
+            evaluatedItemId: { attribute: "evaluated-item-id", type: String },
+            evaluatedItemOwnerId: { attribute: "evaluated-item-owner-id", type: String },
+            summarytype: {attribute: "summarytype", type:String},
+            // Non attribute
+            evaluation: { type: Object },
+            totalPoints: Number,
+            translatedTotalPoints: { type: Number },
+            selectedRatings: { type: Array },
+            criteria: { type: Array },
+            rubric: { type: Object },
+            allEvaluations: { type: Array },
+            columncounter: { type: Number }
```

Comment:
You can add attribute: false to all these. It prevents lit from trying to reflect them to the tags at any point.